pyproject.toml

```diff
@@ -25,11 +25,12 @@ pinecone-text = {version = "^0.7.1", optional = true}
 fastembed = {version = "^0.1.3", optional = true, python = "<3.12"}
 torch = {version = "^2.1.2", optional = true}
 transformers = {version = "^4.36.2", optional = true}
+llama-cpp-python = "^0.2.28"
```

Comment:
need to make `optional = True` otherwise is still a required package (I can modify)